pinot-core/src/main/java/org/apache/pinot/core/operator/BaseOperator.java

```diff
@@ -52,4 +52,14 @@ public final T nextBlock() {
 
   // Make it protected because we should always call nextBlock()
   protected abstract T getNextBlock();
+
+  @Override
+  public String toExplainString() {
+    return getExplainPlanName();
+  }
+
+  @Override
+  public ExecutionStatistics getExecutionStatistics() {
```

Comment:
This method override can be removed because it is the same as the default implementation in the interface

Reply:
Done.